feature-libs/product-configurator/rulebased/components/overview-price/configurator-overview-price.component.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import {
+  ConfiguratorOverviewPriceComponent,
+  OverviewPriceType,
+} from './configurator-overview-price.component';
+
+const mockProductPrice = 100;
+const mockProductQuantity = 10;
+const mockCalculatedTotal = 1000;
+
+describe('ConfiguratorOverviewPriceComponent', () => {
+  let component: ConfiguratorOverviewPriceComponent;
+  let fixture: ComponentFixture<ConfiguratorOverviewPriceComponent>;
+
+  beforeEach(
+    waitForAsync(() => {
+      TestBed.configureTestingModule({
+        declarations: [ConfiguratorOverviewPriceComponent],
+      }).compileComponents();
+    })
+  );
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ConfiguratorOverviewPriceComponent);
+    component = fixture.componentInstance;
+  });
+
+  it('should be created', () => {
+    expect(component).toBeTruthy();
+  });
+
+  describe('overview price type', () => {
+    it('should be PRICE_ONLY', () => {
```

Comment:
rather `it('should be PRICE_ONLY' if component carries product price')`,
To make clear what the test really expects under certain conditions. Similar for other test descriptions

Reply:
Done